comscore/src/main/java/com/bitmovin/player/integration/comscore/ComScoreAnalytics.java

```diff
@@ -7,9 +7,13 @@
 import com.comscore.Analytics;
 import com.comscore.PublisherConfiguration;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public class ComScoreAnalytics {
     private static final String TAG = "ComScoreAnalytics";
     private static boolean started;
+    private static ComScoreConfiguration configuration;
```

Comment:
do we need to store this object? Could we just store the userConsent instead?

Reply:
we do need it as we need to access the `publisherId` (to get the publisher config) as well as the `userConsent`

Reply:
sounds good, lets keep it then